### DISCUSSION TITLE
Testing new method to run ads (Experiment)

### DISCUSSION THREAD

**Discussion by TimilehinO** [DaBible-Foundation/DaBible-MainHub#293] [General]
<img width="1904" alt="Screen Shot 2021-08-05 at 11 28 21 PM" src="https://user-images.githubusercontent.com/80595594/128456072-efaee3e2-b04a-4ac2-9f28-fc23ce6af6d0.png">

So in one of our scrum meetings this week, @emmanlex mentioned why don't we run three ads for one blog post to generate more traffic. To be honest I was not for it, because it sounds so redundant. I said I would give it a shot though. So in the image above, I created an ad group called Pastor Gideon and in that ad group, I have three ads. Now it is time to test the hypothesis and see where we land.

**Comment by sanmiajanaku** [comment]
@***@***.*** ***@***.***> What you are
doing is wrong and lazy. Each ad group mut have at least 3 different Ads
that will be rotated. You have always been running a single ad from a Ad
Grroup, and we have been trying to correct you but you listened.

You are going through this all very wrong.

Thanks,
*Sanmi Ayotunde Ajanaku*
Team Leader - DaBible Foundation
www.dabible.com

**Comment by emmanlex** [comment]
what do you think @TimilehinO
